pyproject.toml

```diff
@@ -862,6 +862,7 @@ module = [
     "sentry.models.organizationmembermapping",
     "sentry.models.organizationmemberteam",
     "sentry.models.organizationonboardingtask",
+    "sentry.models.orgauthtoken",
```

Comment:
the blocklist should not be extended.  please fix the type errors in your new module

Reply:
Ah, I think with https://github.com/getsentry/sentry/pull/50730 this should work (that was the problem that came up)

Reply:
this is gone now!